### DISCUSSION TITLE
leather what is this

### DISCUSSION THREAD

**Discussion by Refenger** [Leather128/LeathersFunkinEngine#301] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/121453687/218072112-ae4da5a1-ff0e-4747-83a6-b60ca4d4612a.png)

**Comment by Refenger** [comment, marked as answer]
oh it's an "The character " + event[3] + " isn't in any character cache!\nHow did this happen? ¯|_(ツ)_|¯" error message

**Reply by Leather128** [reply]
ohhhhhhhh yeah i forgor funny characters hold on
